### PR TITLE
#1982 - Add request/offer label to post, UI and accessibility improvements

### LINKED ISSUE #1982: Add Request/Offer label to post card + accessibility improvements
Following this design: https://www.figma.com/file/EMKcg0eTnOlr7bRZGFdnZu/Web-app-product-spec?node-id=145%3A328

Add the "Request/Offer" label depending on the type of post.
Move the timestamp up to the same line and have the blue dot separating in between label and time.

Accessibility:
-
- Add colour DARKISH_GRAY: #53556E.

Post Card:
- Timestamp and location text colour in the post card should be darkishGray.

Login.js:
- `SectionDiv` **color** property should use darkishGray, same with **border-top** within :before&:after.
- `ButtonText` **color** should also be darkishGray.

### PR DESCRIPTION
**Pull request by CamilaArias**
### What does this pull request aim to achieve? 💯

_Please be concise and link any related issue(s):_
closes: #1982 

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`**, or **`hotfix/<branch_name>`** if this is a hotfix, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [x] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
- [x] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)

### CODE REVIEW

File: client/src/pages/Login.js [hunk 1]
```diff
@@ -544,27 +544,25 @@ const Login = ({ isLoginForm, forgotPassword, isAuthenticated }) => {
                     </p>
                   </>
```
Comment (ddveloper) [inline client/src/pages/Login.js]: changes below are actually `eslint` formatting, only spacing changed but `git diff` is showing it this way, so below changes are all good

**Review by rabbi08 — CHANGES_REQUESTED** [review]
@CamilaArias 
The post card is updated is
- Help Board
- User profile page

but it is not updated on the post page.
Please see below GIF
ezgif com-video-to-gif

**Review by rabbi08 — DISMISSED** [review]
Verified the fix as working fine. the labels are updated and displayed on all the post pages:
- Help Board
- Profile page
- Post page

Thanks @CamilaArias 

Screen Shot 2020-11-30 at 3 27 39 PM

**Review by rabbi08 — APPROVED** [review]
Verified the 'Create post' button, having large '+' sign 
- On Help board
- Individual profile
- Org Profile page

Thanks @CamilaArias 

refer below screenshot
Screen Shot 2020-11-30 at 5 11 08 PM

**Review by Zaydme — APPROVED** [review]
LGTM! Thanks @CamilaArias 🙌

**Pull request merged**